syntax/index.html

```diff
@@ -554,28 +544,29 @@
-          <li>If <code>UM<sub>T</sub></code> does not describe a <a>table</a> or <a>group of tables</a> describing the tabular data file, retrieve the first metadata file (<code>FM</code>) as described in <a href="#locating-metadata" class="sectionRef"></a>:
+          <li>Retrieve the first metadata file (<code>FM</code>) as described in <a href="#locating-metadata" class="sectionRef"></a>:
             <ol class="algorithm">
+              <li>metadata supplied by the user (see <a href="#overriding-metadata" class="sectionRef"></a>).</li>
               <li>metadata referenced from a <em>Link Header</em> that may be returned when retrieving the tabular data file (see <a href="#link-header" class="sectionRef"></a>).</li>
               <li>file-specific metadata in a document located based on the location of the tabular data file (see <a href="#standard-file-metadata" class="sectionRef"></a>).</li>
               <li>directory-specific metadata in a document located based on the location of the tabular data file (see <a href="#standard-directory-metadata" class="sectionRef"></a>).</li>
-              <li><a>default metadata</a> as defined in <a href="#locating-metadata" class="sectionRef"></a> with a single <code>tables</code> entry where the <code>url</code> property is set from that of the <a>tabular data file</a>.</li>
+              <li><a>embedded metadata</a> as defined in <a href="#embedded-metadata" class="sectionRef"></a> with a single <code>tables</code> entry where the <code>url</code> property is set from that of the <a>tabular data file</a>.</li>
             </ol>
           </li>
-          <li>Proceed as if the process starts with a metadata file using the merged metadata <code>UM<sub>M</sub> = merge(UM<sub>T</sub>, FM)</code> or <code>UM<sub>T</sub></code> if no <code>FM</code> is loaded.</li>
+          <li>Proceed as if the process starts with <code>FM</code>.</li>
         </ol>
         <p>If the process starts with a metadata file:</p>
         <ol class="algorithm">
-          <li>Retrieve the metadata file yielding the metadata <code>UM<sub>M</sub></code> (which is treated as overriding metadata, see <a href="#overriding-metadata" class="sectionRef"></a>).</li>
-          <li>For each <a>table</a> in <code>UM<sub>M</sub></code> in order, create one or more <a title="annotated table">annotated tables</a>:
+          <li>Retrieve the metadata file yielding the metadata <code>UM</code> (which is treated as overriding metadata, see <a href="#overriding-metadata" class="sectionRef"></a>).</li>
+          <li>Normalize <code>UM</code> using the process defined in <cite><a href="http://w3c.github.io/csvw/metadata/#normalization">Normalization</a></cite> in [[!tabular-metadata]], coercing <code>UM</code> into a <a href="http://w3c.github.io/csvw/metadata/#dfn-table-group-description" class="externalDFN">table group description</a>, if necessary.</li>
```

Comment:
This is a section reference, not a _notion_. IMO, the property descriptions should have probably been dfns rather than simple anchors, to make this clear; in this case, it's wrapped in a _cite_ element, which is my convention, at least, when referencing a section in another document, along with the actual document reference.